chord_metadata_service/chord/api_views.py

```diff
@@ -117,6 +120,15 @@ async def destroy(self, request, *args, **kwargs):
         authz.mark_authz_done(request)
         return await sync_to_async(super().destroy)(request, *args, **kwargs)
 
+    @async_to_sync
+    async def list(self, request, *args, **kwargs):
```

Comment:
there are a few more endpoints for viewing project/dataset metadata (a bunch of public endpoints + maybe some old CHORD endpoints too), I think. Also, how will this interface with Beacon?